### DISCUSSION TITLE
about duplicate vertex and edge

### DISCUSSION THREAD

**Discussion by wuyueandrew** [alibaba/GraphScope#1845] [General]
consider a bench dataset like：

vertex label a:
```
aid
1
2
3
3
```

vertex label b:
```
bid
6
6
7
```

edge(a -> b)
```
src_id,dst_id
1,6
2,6
3,7
```

```
graph = graph.add_vertices('/tmp/dataset_test/duplicate_dataset/a.csv', label='a', vid_field='aid')
graph = graph.add_vertices('/tmp/dataset_test/duplicate_dataset/b.csv', label='b', vid_field='bid')
graph = graph.add_edges('/tmp/dataset_test/duplicate_dataset/a_to_b.csv',
                        src_label='a',
                        dst_label='b')
interactive = sess.gremlin(graph)

node_num = interactive.execute("g.V().count()").one()
edge_num = interactive.execute("g.E().count()").one()
```

What's the reason of node_num=7, edge_num=3 ?
I think `node_num=7` and `edge_num=3` are confilct.
1.Duplicate vertexs which have equal vid_field can be create repeatly ? 
Is vid_field a unique key and gs has a oid like primary key?
2.If the answer 1 is yes, the edge number should be 5 (2 * (1->6) + 2 * (2->6) + 1 * (3 ->7)) i thought.
what's the reason of edge_num = 3 (due to this, duplicate vertexs which have equal vid_field can not be created obviously)?

**Comment by yecol** [comment]
Answered in #1877
